lib/nexpose/scan_template.rb

```diff
@@ -583,5 +583,20 @@ def enable_enhanced_logging=(enable)
       self.enable_debug_logging = enable
       self.aces_level = (enable ? 'full' : 'none')
     end
+
+    # Enable or disable windows service editor
+    # @param [Boolean] enable Enable or disable windows service editor
```

Comment:
in `@param [Boolean] enable Enable or disable` the 'enable' is name of parameter where 'Enable or disable' is description, is it not?